tests/test_triggers.py

```diff
@@ -99,3 +99,8 @@ def test_http_trigger_extra():
     http_trigger = HttpTrigger(annotations={"x": "123"}, extra_attributes={"y": "456"})
     assert http_trigger._struct["annotations"]["x"] == "123"
     assert http_trigger._struct["attributes"]["y"] == "456"
+
+
+def test_renamed_deprecated_values():
+    trigger = V3IOStreamTrigger(seekTo="test", access_key="abc")
+    assert trigger._struct["attributes"]["seekTo"] == "test"
```

Comment:
This test is not ideal, because it passes even without your changes.

Reply:
@gtopper exactly this we would like to check, because we want to be backward compatible

Reply:
So check `accessKey` as well :)

Reply:
@TomerShor I did nothing with `accessKey` (it is just a must-have arg)

Reply:
Maybe then we could check both the old parameters and new ones? Either in the same test or a separate one.

Reply:
@gtopper added